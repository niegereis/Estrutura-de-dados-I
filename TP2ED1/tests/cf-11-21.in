11 21
f
*********************
*   * *     *       *
* * * * *** * ***** *
* *   * * *   *M    *
* * *** * ***********
* * *   *           *
* * * *** ***** *** *
* * * *       * * * *
* *** ********* * * *
*               *    
*********************
