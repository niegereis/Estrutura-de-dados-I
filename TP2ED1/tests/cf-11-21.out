89
*********************
*ooo*o*ooooo*ooooooo*
*o*o*o*o***o*o*****o*
*o*ooo*o*o*ooo*Moooo*
*o*o***o*o***********
*o*o*ooo*ooooooooooo*
*o*o*o***o*****o***o*
*o*o*o*  o    *o* *o*
*o***o*********o* *o*
*ooooooooooooooo*  oo
*********************
